### ISSUE TITLE
TensorFlow implementation of scVI

### ISSUE THREAD

**Issue by romain-lopez** [vals/Blog#2]
Hi Valentine,

Thank you so much for your interest in scVI and for your blog article. The comparison is very interesting and valuable. I'd like to make a few comments:

1 "And even without it, it seems to align the two mice quite well in the Lukassen 2018 data!" -> This should be because scVI performs implicit normalization inside its graphical model.
2 scVI also performs hypothesis testing, aka differential expression (because it is a fully generative model and not exactly an autoencoder)
3 I'm confused about the running time. Whatever the hardware, scVI does essentially very similar computations than DCA does. It is possible that on the dataset you have, less epochs might be needed. 
4 We are working hard to deliver a PyTorch package to the public that would make testing easier for everyone (as you exchanged with Jeffrey a few hours ago). Coming soon !

Best, 
Romain
